toy_example/README.md

```diff
@@ -0,0 +1,62 @@
+# Toy example
+
+Here we present a walkthrough of running `pandora` on a toy example. We run: 1) `pandora` without de novo discovery; 2) `pandora` workflow (https://github.com/iqbal-lab-org/pandora_workflow), which runs pandora with and without de novo discovery (see Figure 2 of Colquhoun et al. "Nucleotide-resolution bacterial pan-genomics with reference graphs"). Although method 2) run both modes of `pandora`, it is much more involved than method 1), as the user needs to configure and run a `snakemake` pipeline. If no de novo discovery is required, method 1 is a lot simpler to run (two command lines) as opposed to running the pipeline. For completeness, we show both methods.
+
+## Input data description
+
+```
+msas/ : contains the MSAs of the two genes we are using as toy example here, GC00006032 and GC00010897;
+prgs/toy_prg.fa : contains the PRGs of the two genes, GC00006032 and GC00010897. These are fairly simple PRGs. GC00006032 contains 4 variant sites, each representing a SNP, while GC00010897 contains 5;
+reads/ : contains perfect simulated reads from two toy samples. We simulated perfect reads, where one sample genotypes to one allele of the variant sites, while the other sample genotypes towards the other allele;
+pandora_workflow_data : contains other input and configuration files to run the pandora workflow;
+```
+
+##  pandora without de novo discovery
+
+### Dependencies
+
+* `singularity`
+
+### Running
+`cd toy_example && ./run_pandora_nodenovo.sh`
```

Comment:
Could you put this in a shell code fence? I tried doing it in the suggestions but it screws it up due to the code fence escaping the suggestion